invenio_drafts_resources/services/records/service.py

```diff
@@ -527,6 +529,16 @@ def _validate_draft(self, identity, draft):
             # explicit
         )
 
+        if draft.files.enabled:
+            for file_record in draft.files.values():
+                if not TransferType(file_record.file.storage_class).is_completed:
```

Comment:
To be checked, is `file_record.file.storage_class` already of class TransferType? (check `__class__` since the `__repr__` will show a [string](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/files/transfer.py#L39))

Reply:
So doing 
`print(type(file_record.file.storage_class))`
I get
`<class 'str'>
`
So the class instantion is needed no?

Reply:
Yup, you are right! thanks for checking!